deployment/ion/ion.tf

```diff
@@ -129,6 +129,145 @@ resource "kubernetes_deployment" "ion-front-api" {
   }
 }
 
+resource "tls_private_key" "ca" {
+  algorithm = "RSA"
+  rsa_bits  = "2048"
+}
+
+resource "tls_self_signed_cert" "ca" {
```

Comment:
Nit: Maybe the CA/Cert stuff should be in a /cert module rather than under the ion one.